pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2015 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build cgo,linux
+
+package cadvisor
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/golang/glog"
+	cadvisorHttp "github.com/google/cadvisor/http"
+	cadvisorApi "github.com/google/cadvisor/info/v1"
+	"github.com/google/cadvisor/manager"
+	"github.com/google/cadvisor/storage/memory"
+	"github.com/google/cadvisor/utils/sysfs"
+)
+
+type cadvisorClient struct {
+	manager.Manager
+}
+
+var _ Interface = new(cadvisorClient)
+
+// TODO(vmarmol): Make configurable.
+// The number of stats to keep in memory.
+const statsToCache = 60
```

Comment:
Can this be time duration instead of a number? Since we have dynamic housekeeping, a time duration make more sense. This is probably a comment for cadvisor in general.